CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+
+## 2.28.0-rc.0 / 2021-06-17
+
+[CHANGE]: UI: Make the new experimental PromQL editor the default. #8925
+[FEATURE]: Linode Discovery: Add Linode service discovery. #8846
+[FEATURE]: HTTP Discovery: Add generic HTTP-based service discovery. #8839
```

Comment:
```suggestion
[FEATURE]: HTTP SD: Add generic HTTP-based service discovery. #8839
```

Will be better communication wise.

Reply:
I just used `Discovery` vs. `SD` everywhere because I saw previous changelogs do that. Should I maybe use `SD` everywhere? Or just for the HTTP one?

Reply:
Both were used in the past. e.g. 2.21 has a bunch of SD.

Reply:
Changed everything to say "SD" in the prefixes.